godtools/push/action.yaml

```
@@ -14,14 +14,18 @@ inputs:
     required: false
     default: acr
     description: List of names of the registries where an image should be pushed (ecr, acr, docker-hub, internal-acr).
+  tag-latest:
+    required: false
+    default: "true"
+    description: Prepare desired tag from latest image before pushing it to the registry
 runs:
   using: "composite"
   steps:
     - name: Push tmp docker image
       if: "!startsWith(github.ref, 'refs/tags')"
       shell: bash
       run: |
-        docker tag ${{ inputs.image }} ${{ inputs.image }}:tmp-$(git rev-parse --short HEAD) && 
+        ["${{ inputs.tag-latest }}" = "true"] && docker tag ${{ inputs.image }} ${{ inputs.image }}:tmp-$(git rev-parse --short HEAD); 
```

```suggestion
        ["${{ inputs.tag-latest }}" == "true"] && docker tag ${{ inputs.image }} ${{ inputs.image }}:tmp-$(git rev-parse --short HEAD);
```

Currently there is a syntax error (38th line as well). I think it needs double `=`